sdb/commands/stacks.py

```diff
@@ -52,6 +52,9 @@
 # future iterations.
 #
 class Stacks(sdb.Command):
+    """
+    Print the stack traces for the current threads
```

Comment:
I think it would be great if we had some examples of using `stacks -m module` and `stacks -c function` here!

Reply:
OK, I'll work on that

Reply:
Are you planning to add these examples in this PR?